src/app/shared/conditional-show/conditional-show.directive.ts

```diff
@@ -100,6 +100,13 @@ export class ConditionalShowDirective implements AfterViewInit, OnDestroy {
 
   private show() {
     this.el.nativeElement.hidden = false;
+    if (this.caseField && this.caseField.field_type && this.caseField.field_type.type !== 'Collection') {
+      if (this.el.nativeElement.closest('ccd-write-complex-type-field')) {
+        this.el.nativeElement.children[0].children[0].classList.add('show-condition-gray-bar');
+      } else {
+        this.el.nativeElement.classList.add('show-condition-gray-bar');
+      }
+    }
```

Comment:
The intent of the whole code block above is lost in the complexity of the conditions and deep object navigation I think. Introducing private method and/or intermediary variables would make it more meaningful.

Reply:
Done.